### ISSUE TITLE
nd4j-native - (Windows?) Snapshots - ClassNotFoundException

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/nd4j#2720]
I haven't seen this before when building locally, but when using snapshots (from sonatype), I run into this. Seems to occur on all of the examples... (examples branch: https://github.com/deeplearning4j/dl4j-examples/pull/589)

Note that this doesn't actually stop me from running the examples (at least those that I've tried so far).

https://gist.github.com/AlexDBlack/2fb2677225a78cbf7657ffcba2549463

Specific snapshot versions:
nd4j-native-0.9.2-20180308.203333-2395.jar
nd4j-native-0.9.2-20180308.193237-2388-windows-x86_64.jar

Edit: This definitely does not occur when building locally instead.

**Comment by saudet** [comment]
@raver119 Are these classes disabled on Windows by any chance?

**Comment by raver119** [comment]
AFAIK this issue isn't relevant anymore, and probably was caused by desync between libnd4j and nd4j.

**Comment by AlexDBlack** [comment]
Nope, definitely is still relevant. And @maxpumperla reported seeing it on Mac, at least once.

**Comment by saudet** [comment]
Yes, that won't happen with a monorepo...

**Comment by sshepel** [comment]
Hey @AlexDBlack I have checked build logic on Jenkins and didn't see any excludes or smth like that for missing package neither for windows platform nor any other platform.

Unfortunately, the snapshot version that you have provided has already been removed from nexus, and I can't say if missing classes were in jar, but I've check the latest snapshot and can clearly see that all classes are there (see listing for the jar [here](https://gist.github.com/sshepel/d0870a216bc84ae46c7977d20ffe0305))
